Dockerfile

```diff
@@ -28,4 +28,5 @@ RUN apk --update add bash fuse libcurl libxml2 libstdc++ libgcc alpine-sdk autom
 RUN sed -i s/"#user_allow_other"/"user_allow_other"/g /etc/fuse.conf
 
 COPY docker-entrypoint.sh /
+RUN ["chmod", "+x", "/docker-entrypoint.sh"]
```

Comment:
Would it be better to use:

```
COPY --chmod=0755 docker-entrypoint.sh /
```

Reply:
I think the better way is to set file permission for `docker-entrypoint.sh` :

```
chmod +x docker-entrypoint.sh
```

----

`COPY` in Dockerfile will keep file permission. There is no further action if the file permission has set correctly.